charts/hdfs-k8s/templates/datanode-daemonset.yaml

```diff
@@ -0,0 +1,85 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: resolv-conf-datanode
+  namespace: kube-system
+data:
+  resolv.conf: |
+    search kube-system.svc.{{ .Values.clusterDomain }} svc.{{ .Values.clusterDomain }} {{ .Values.clusterDomain }}
+    nameserver {{ .Values.clusterDnsIP }}
+    options ndots:5
+---
+# Deleting a daemonset may need some trick. See
+# https://github.com/kubernetes/kubernetes/issues/33245#issuecomment-261250489
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: hdfs-datanode
+  namespace: kube-system
+spec:
+  template:
+    metadata:
+      labels:
+        name: hdfs-datanode
+      annotations:
+        scheduler.alpha.kubernetes.io/tolerations: |
```

Comment:
Hmm. So it seems this annotation specifically allows the master node to run a member daemon. I dropped it because that's not what we wanted.

But a daemon is still scheduled on the master node. I think I read about this behavior as a bug. Anyway, not having this annotation is better. So I'll update the patch.